packages/graphql-transformers/src/openCrudWhereTransform/manyRelationConditionSubquery.js

```diff
@@ -0,0 +1,72 @@
+const Sequelize = require('sequelize');
+const { trimEnd, lowerFirst } = require('lodash');
+
+const Op = Sequelize.Op;
+
+function buildConditionSubquery(targetModel, entityName, relatedEntityFilter, isNegative = false) {
+  const positiveOrNegative = isNegative ? Op.not : Op.and;
+  const queryGenerator = targetModel.QueryGenerator;
+  let associationKey = Object.keys(targetModel.associations).find(k => targetModel.associations[k].target.name === entityName);
+  let association = targetModel.associations[associationKey];
+  let selectQuery;
+  if (association.associationType === 'BelongsToMany') {
```

Comment:
I'd make them all consts somewhere (hasOne, belongsTo, belongsToMany etc), just cause it looks nicer

Reply:
done